storyblok/field-plugin
# EXT-1031: Replaced actions and effects module with createFieldPlugin module

## What?

Replaced actions and effects module with createFieldPlugin module

- Moved all types in the actions module to the `effects/createFieldPlugin` module
- removed the now empty actions module
- Moved the `/effects/createFieldPlugin` to `/createFieldPlugin`
- Removed the now empty `/effect` module

## Why?

- The actions module contained just a single file, after the `postX` functions were removed in #26. So it makes sense to move that logic into the `effects` module, where it was being used.
- The effects module contained just two exports; `createFieldPlugin` and `PluginState`, so it makes sense to replace the `effects` module with `createFieldPlugin`.

## How to test? (optional)

Everything will work as before

```shell
yarn
yarn lint
yarn check:types
yarn build && yarn build
```
